src/actions/index.ts

```diff
@@ -1,25 +1,48 @@
-import {Action} from 'redux';
+import {
+  TValuePayload,
+  TUpdateInArray,
+  TIndexPayload,
+  TSaveForm
```

Comment:
We're using `I` for types elsewhere, is there a reason for using `T` here instead?